docker/php/Dockerfile

```diff
@@ -1,32 +1,60 @@
-FROM php:5-fpm
+FROM php:5.6-fpm
 
-RUN apt-get update -qq \
-  && apt-get install wget mysql-client -yq
+# Install additional tools:
+RUN apt-get update -qq && apt-get install -yq \
+        mysql-client \
+        nano \
+        vim \
+        wget
 
-# install the PHP extensions we need
-RUN apt-get update && apt-get install -y libpng12-dev libjpeg-dev libpq-dev \
+# install the PHP extensions we need:
+RUN apt-get update -qq && apt-get install -yq \
+        libpng12-dev \
+        libjpeg-dev \
+        libpq-dev \
     && rm -rf /var/lib/apt/lists/* \
     && docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr \
-    && docker-php-ext-install gd mbstring opcache pdo pdo_mysql pdo_pgsql zip
-# set recommended PHP.ini settings
-# see https://secure.php.net/manual/en/opcache.installation.php
-RUN { \
```

Comment:
I originally took this from the official docker image, it was required. Did this moved somewhere else?